lib/src/kvs/ds.rs

```diff
@@ -213,6 +213,18 @@ impl Datastore {
 					info!("Starting kvs store at {}", path);
 					let s = s.trim_start_matches("file://");
 					let s = s.trim_start_matches("file:");
+
+					let s = match s.starts_with('~') {
+						true => s.replacen(
+							'~',
+							dirs::home_dir().unwrap().as_os_str().to_str().unwrap(),
```

Comment:
Rewritten as if statement to check before unwrapping. That way panic will be ruled out. Using the dirs crate instead of check for env variable `HOME` to ensure equal behaviour on all platforms